src/olgm/gm.js

```diff
@@ -475,11 +476,12 @@ export function createLabel(textStyle, latLng, index) {
  * @param {number} index index for the label
  * @return {module:olgm/gm/MapIcon} map icon
  */
-export function createMapIcon(iconStyle, latLng, index) {
+export function createMapIcon(iconStyle, latLng, index, pane) {
 
   const iconOptions = {
     align: 'center',
     position: latLng,
+    pane: pane,
```

Comment:
here, the `iconOptions` are native to OpenLayers.  `pane` is not.  We should reuse the same name as above (i.e. olgm_pane).

Reply:
Changed.